Tribler/Test/Core/Modules/MetadataStore/test_store.py

```diff
@@ -93,6 +93,7 @@ def test_process_channel_dir_file(self):
 
     @db_session
     def test_squash_mdblobs(self):
+        random.seed(123)
```

Comment:
Note that this will affect the randomness of all tests. It's better to create an instance of `Random` and seed/use that one.